locales/kab_DZ/LC_MESSAGES/duckduckgo.po

```diff
@@ -3304,12 +3301,12 @@ msgstr "Tabniqt n unadi"
 
 msgctxt "settings"
 msgid "Search Button Background"
-msgstr ""
+msgstr "Agilal n tqeffalt n unadi"
 
 #. search for a place within the map area currently shown in expanded map
 msgctxt "vertical_map"
 msgid "Search Current Map Area"
-msgstr ""
+msgstr "Anadi deg temnaḍt tamirant"
 
 msgid "Search DuckDuckGo"
 msgstr "Nadidi DuckDuckGo"
```

Comment:
"Nadidi DuckDuckGo" should be "Nadi di DuckDuckGo" ?

Reply:
"Nadi di DuckDuckGo"

Reply:
Tanemmirt !